source/_integrations/habitica.markdown

```
@@ -296,6 +296,25 @@ Updates a specific reward for the selected Habitica character.
+### Action `habitica.update_habit`
+
+Updates a specific habit for the selected Habitica character.
+
+| Data attribute | Optional | Description                                                                                  |
+| -------------- | -------- | -------------------------------------------------------------------------------------------- |
+| `config_entry` | no       | Select the Habitica account to update a habit.                                               |
+| `task`         | no       | The name (or task ID) of the habit you want to update.                                       |
+| `rename`       | yes      | The new title for the Habitica habit.                                                        |
+| `notes`        | yes      | The new notes for the Habitica habit.                                                        |
+| `up_down`      | yes      | Update if the habit is good and rewarding (positive), bad and penalizing (negative) or both. Valid values: `up`, `down` |
+| `priority`     | yes      | Update the difficulty of a habit. Valid values: `trivial`, `easy`, ` 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)